src/item-navigator.ts

```diff
@@ -368,6 +368,15 @@ export class ItemNavigator
         display: block;
       }
 
+      :host,
+      #frame,
+      .menu-and-reader,
+      .loading-area,
+      .loading-view {
+        min-height: inherit;
+        height: inherit;
+      }
```

Comment:
What is the `inherit` doing in this case.. or maybe what happens if you don't inherit?

Reply:
inherit is just passing the externally set height down to the item-nav components.
without it, its height is 0.